src/controllers/authentication.controller.js

```diff
@@ -0,0 +1,30 @@
+import { User } from "../database/models";
+import bcrypt from "bcrypt";
+import jwt from "jsonwebtoken";
+import { TOKEN_SECRET } from "../config";
+
+export const signIn = async (req, res) => {
+  try {
+    let user = await User.findOne({ where: { email: req.body.email } });
```

Comment:
Outsource this user function and create it out in service so that anyone else can use it anywhere